package com.jonathansandell.expense_splitter.repository;

import com.jonathansandell.expense_splitter.entity.Member;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface MemberRepository extends JpaRepository<Member, Long> {

    List<Member> findByGroupId(Long groupId);
}
